docs/v1/P4Runtime-Spec.mdk

```diff
@@ -3955,6 +3976,14 @@ stream using two configured rates --- the Peak Information Rate (PIR) and
 Committed Information Rate (CIR), and their associated burst sizes --- and
 "marks" its packets as GREEN, YELLOW or RED based on the observed rate.
 
+`Meter` & `DirectMeter` have an additional field `counter_data` that may hold
+per color counter data. It has a granular list of the counters applicable to the
+possible meter colors such as GREEN, YELLOW and RED. The counter associated with
```

Comment:
Nit: remove "such as", since GREEN, YELLOW, RED is the exact list of colors we support.